packages/components/buttons/primary-button/README.md

```diff
@@ -57,7 +57,7 @@ export default Example;
 | `isToggled`      | `boolean`                                                        |          |             | Tells when the button should present a toggled state. It does not have any effect when `isToggleButton` is `false`.                                                                                                                                                         |
 | `isDisabled`     | `boolean`                                                        |          |             | Tells when the button should present a disabled state.                                                                                                                                                                                                                      |
 | `onClick`        | `Function`<br/>[See signature.](#signature-onClick)              |          |             | Handler when the button is clicked.&#xA;<br />&#xA;Required when `as` is `undefined`                                                                                                                                                                                        |
-| `size`           | `union`<br/>Possible values:<br/>`'medium' , 'big'`              |          | `'big'`     | Indicates the size of the button.                                                                                                                                                                                                                                           |
+| `size`           | `union`<br/>Possible values:<br/>`'small' , 'medium' , 'big'`    |          | `'big'`     | Indicates the size of the button.                                                                                                                                                                                                                                           |
```

Comment:
If we want to introduce the `small` value here, we should update the description to state it is deprecated which can be done with the JSDoc of the `size` property in the `TPrimaryButtonProps` type.

Reply:
@CarlosCortizasCT I'm not sure I follow, sorry. Could you explain some more, please?

Reply:
I guess we don't want the 'small' size to be available here. But how do I do that?

Reply:
Based on [what we announce in the release note](https://github.com/commercetools/merchant-center-application-kit/blob/595d65ae947ed38531f3b7f3b698b7b96931da2f/website/src/releases/2023-03/button-sizes.mdx?plain=1#L14-L17) and what Carlos suggested we could edit JSDoc [here](https://github.com/commercetools/ui-kit/blob/12d05bc502490f271e4255463f2e226af24387bd/packages/components/buttons/primary-button/src/primary-button.tsx#L62-L65) and add something like:

```ts
/**
 * Indicates the size of the button.
 * <br />
 * @deprecated `small` is deprecated. Use `medium` instead.
 *
 */
size?: 'small' | 'medium' | 'big';
```
and re-generate the readme.

Reply:
Yes @kark that was exactly that. Thank you! 🙏 Carlos jumped in the huddle with me to quickly explain it to me how the README files are working.

Reply:
Here is the change - [4cf13fe](https://github.com/commercetools/ui-kit/pull/2455/commits/4cf13fe90d5751e354918b7fb63c56b0480d82b3).